test/test_mimic_explainer.py

```diff
@@ -237,17 +237,18 @@ def test_explain_model_categorical(self, verify_mimic):
         for verifier in verify_mimic:
             verifier.verify_explain_model_categorical(pass_categoricals=True)
 
-    @pytest.mark.skip(reason='Pandas indexes not supported yet')
     @pytest.mark.parametrize("sample_cnt_per_grain,grains_dict", [
         (240, {}),
         (20, {'fruit': ['apple', 'grape'], 'store': [100, 200, 50]})])
     def test_dataframe_model(self, mimic_explainer, sample_cnt_per_grain, grains_dict):
         X, _ = create_timeseries_data(sample_cnt_per_grain, 'time', 'y', grains_dict)
         model = DataFrameTestModel(X.copy())
         model = Pipeline([('test', model)])
-        features = list(X.columns)
+        features = list(X.columns.values) + list(X.index.names)
         model_task = ModelTask.Unknown
-        kwargs = {'explainable_model_args': {'n_jobs': 1}, 'augment_data': False}
+        kwargs = {'explainable_model_args': {'n_jobs': 1}, 'augment_data': False, 'reset_index': True}
+        if grains_dict:
+            kwargs['categorical_features'] = ['fruit']
```

Comment:
Doesn't seem like nearly enough testing for the amount of custom code that was written

Reply:
added more tests